### PR TITLE
LPS-145404 Check if transition default exists to set the defaultEdge

### PR DESCRIPTION
**Pull request by alinedoleron**
Ticket: https://issues.liferay.com/browse/LPS-145404

https://user-images.githubusercontent.com/26491796/151458900-2579b9a1-8b96-43b7-b7a8-9b6b2fa7b99e.mp4

### CODE REVIEW

**Comment by alinedoleron** [comment]
ci:test:relevant

**Review by naraandrade — CHANGES_REQUESTED** [review]
Check if there's already a transition with defaultEdge == true in the transition's source node

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
> Check if there's already a transition with defaultEdge == true in the transition's source node

Hi @naraandrade, I've made the changes.

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/source-builder/deserializeUtil.js [hunk 1]
```diff
@@ -121,10 +123,21 @@ DeserializeUtil.prototype = {
...
+								isEdge(element) &&
+								element.source === nodeId &&
+								element.data.defaultEdge
+						);
+
 						elements.push({
 							arrowHeadType: 'arrowclosed',
 							data: {
-								defaultEdge: JSON.parse(transition.default),
+								defaultEdge:
```
Comment (naraandrade) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/source-builder/deserializeUtil.js:136]: Update condition

**Review by naraandrade — CHANGES_REQUESTED** [review]
Update

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/source-builder/deserializeUtil.js (hunk 1 above)
Comment (alinedoleron) [reply modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/source-builder/deserializeUtil.js:136]: Thank you, Nara! I've made the changes.

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by naraandrade** [comment]
ci:forward

**Comment by naraandrade** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
ci:forward

**Pull request closed without merging**